packages/core/src/pages/fs-auth-login/fs-auth-login.module.scss

```
@@ -0,0 +1,41 @@
+@layer components {
+  .fsAuthLogin {
+    @import "@faststore/ui/src/components/atoms/Button/styles.scss";
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)